src/docfx/validation/CustomRuleProvider.cs

```diff
@@ -122,9 +122,11 @@ private bool TryGetCustomRule(Error error, [MaybeNullWhen(false)] out CustomRule
                     // compare with code + propertyPath + contentType
                     var source = error.Source?.File;
                     var pageType = source != null ? _documentProvider.GetPageType(source) : null;
-                    if (rule.PropertyPath.Equals(error.PropertyPath, StringComparison.Ordinal) &&
-                        rule.ContentTypes != null &&
-                        rule.ContentTypes.Contains(pageType))
+                    var isPageTypeInScope =
+                        pageType == null ||
+                        rule.ContentTypes == null ||
```

Comment:
Not need to check if `rule.ContentTypes == null` since it is marked as not nullable.

Reply:
no, rule.ContentTypes is nullable